__tests__/components/TableRow.spec.ts

```diff
@@ -0,0 +1,370 @@
+            const vm = wrapper.vm as TableRowTs;
+
+            // Act + Assert:
+            // @ts-ignore
+            expect(vm.hasAvailableActions).toBe(expectedResult);
+            expect(wrapper.find('.pop-tip').exists()).toBe(expectedResult);
+
+            if (expectedResult) {
+                wrapper.find('.poptip-actions').trigger('click');
+                expect(wrapper.emitted('on-show-alias-form')).toBeDefined();
+            } else {
+                expect(wrapper.find('.poptip-actions').exists()).toBe(false);
+            }
+        };
+
+        test('hide Poptip component when assert type is account restrictions', () => {
+            runBasicHasAvailableActionsTests(
+                {
+                    assetType: 'accountRestrictions',
+                    rowValues: {
+                        ...mockNamespaceRowValue,
+                    },
+                },
+                false,
+            );
+        });
+
+        test('hide Poptip component when a namespace is expired', () => {
+            runBasicHasAvailableActionsTests(
+                {
+                    rowValues: {
+                        ...mockNamespaceRowValue,
+                        expiration: 'expired',
+                    },
+                },
+                false,
+            );
+        });
+
+        test('hide Poptip component when an asset does not belong to the user', () => {
+            runBasicHasAvailableActionsTests(
+                {
+                    ownedAssetHexIds: ['0'.repeat(16)],
+                    rowValues: mockMosaicValue,
+                },
+                false,
+            );
+        });
+
+        test('renders Poptip component when user owned asset', () => {
+            runBasicHasAvailableActionsTests(
+                {
+                    ownedAssetHexIds: [mockMosaicValue.hexId],
+                    rowValues: mockMosaicValue,
+                },
+                true,
+            );
+        });
+    });
+
+    describe('isSupplyMutableMosaic', () => {
+        const runBasicSupplyMutableMosaic = (params, expectedResult) => {
+            // Arrange:
+            const wrapper = getTableRowWrapper({
+                ownedAssetHexIds: [mockMosaicValue.hexId],
+                rowValues: {
+                    ...mockMosaicValue,
+                    ...params,
+                },
+            });
+
+            const vm = wrapper.vm as TableRowTs;
+
+            // Act + Assert:
+            // @ts-ignore
+            expect(vm.isSupplyMutableMosaic).toBe(expectedResult);
+            if (expectedResult) {
+                wrapper.find('.modify-supply').trigger('click');
+                expect(wrapper.emitted('on-show-mosaic-supply-change-form')).toBeDefined();
+            } else {
+                expect(wrapper.find('.modify-supply').exists()).toBe(expectedResult);
+            }
+        };
+
+        test('returns true when mosaics is support supply mutable and not expired', () => {
```

Comment:
mosaics is support => mosaic supports(multiple) 
negative: mosaic does not support